dtreeviz/trees.py

```diff
@@ -931,7 +931,8 @@ def rtree_feature_space3D(self,
         just the 2 variables used for training.
         """
         if features is None:
-            features = self.shadow_tree.feature_names[0:2] # pick first two features if none given
+            n_features = len(self.shadow_tree.feature_names)
+            features = self.shadow_tree.feature_names[0:min(n_features,2)] # pick first one/two features if none given
```

Comment:
@parrt wouldn't `_rtreeviz_bivar_3D` crash anyway if `len(features) < 3`? I think it should just raise a `ValueError`.

Reply:
Cool. Improved to check for !=2. How'z it look?

Reply:
LGTM! I cleaned up the comment in a commit.